examples/lusid/instruments/upload-simple-instruments.sql

```diff
@@ -0,0 +1,32 @@
+-- ============================================================
+
+-- 1. In this query, we run an ETL process on some instruments.
+-- 2. First, we load a CSV file of instruments from Drive.
+-- 3. Next, we transform the shape of the instrument data.
+-- 4. Finally we upload the instrument data into LUSID.
+
+-- ============================================================
+
+-- Load file of instruments data
+@instruments_data =
+
+use Drive.Excel
+--file=/luminesce-examples/instruments.xlsx
+--worksheet=simple_instruments
+enduse;
+
+-- Run instruments transformation
+@instruments_for_upload =
```

Comment:
Could this be changed to @simple_instruments

Reply:
Change made.